docs/source/model_design/general_research.rst

```diff
@@ -640,9 +643,184 @@ Cross-Sectional Studies
+
+.. note::
+  This odds ratio is the same as that obtained from a cohort study following exposed and unexposed individuals to see if they develop disease over time. 
+
+
+Rare-disease assumption
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If the study is appropriately designed, in particularly, that sampling of cases and controls is independent of their exposure status, and the disease is rare (incidence of the outcome of interest is <10% in the study population, see :download:`Zhang (1998) What's the Relative Risk? <Zhang_1998_What_is_a_relative risk>` then the odds ratio provides an estimate of the risk ratio or the rate ratio. This is because when the disease is rare in the study population, the odds ratio, risk ratio and rate ratio are all very similar **numerically**.
+
+The risk ratio of a :underline:`cohort study` is given by the following 2x2 table
+
++-----------+---------+----------+----------+
+|           | with    | without  | Total    |
+|           | disease | disease  |          |
++-----------+---------+----------+----------+
+| Exposed   |  a      |  b       | a+b      |
++-----------+---------+----------+----------+
+| Unexposed |  c      |  d       | c+d      | 
++-----------+---------+----------+----------+
+|           | a+c     | b+d      | a+b+d+c  |
++-----------+---------+----------+----------+ 
+
+| Risk of disease in the exposed: :math:`\frac{a}{a+b}` 
+| Risk of disease in the unexposed: :math:`\frac{c}{c+d}`
+| The **risk ratio** is thus :math:`\frac{a/(a+b)}{c/(c+d)}`
+
+If the disease is rare, a and c are very small compared to b and d, and so contribute very little to the denominators (a + b) and (c + d). 
+
+| So RR = :math:`\frac{a/(a+b)}{c/(c+d)}` ~ :math:`\frac{a/b}{c/d}` = :math:`\frac{ad}{bc}`
+
+
+Common-diseases
+~~~~~~~~~~~~~~~
+
+When the disease is common, which measure (risk ratio, rate ratio or odds ratio) the odds ratio estimates will depend on how controls are sampled. 
+
+.. todo:: 
+
+  discuss common disease estimates
+
+
+Analysis for matched study
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ .. todo::
+
+
+PAF for unmatched case-controls
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider a simple unmatched case-control study in which we have a representative sample of cases, together with controls chosen as a random sample of the source population. Then θ can be estimated as usual by the odds ratio, while p or p' can be estimated as the proportion of controls or cases who are exposed:
+
++-----------+---------+----------+----------+
+|           | Cases   | Controls | Total    |
++-----------+---------+----------+----------+
+| Exposed   |  a      |  b       | m_1      |
++-----------+---------+----------+----------+
+| Unexposed |  c      |  d       | m_0      |
++-----------+---------+----------+----------+
+|           | n_1     | n_0      | N        |
++-----------+---------+----------+----------+ 
+
+ | OR = :math:`\frac{ad}{bc}` = θ (relative risk)
+ | p(proportion exposed in the study population) = :math:`\frac{b}{n_0}`
```

Comment:
I think you mean `p` equals "proportion exposed in the control population", and `p'` equals "proportion exposed among cases"

Reply:
corrected